provisioning/dev/docker/Dockerfile

```diff
@@ -18,6 +18,7 @@ COPY Makefile /tmp/build/Makefile
 COPY scripts  /tmp/build/scripts
 RUN cd /tmp/build && make tools && cd / && rm -rf /tmp/build && go clean -cache -modcache
 RUN apt update && apt install -y graphviz
+RUN apt-get install --no-install-recommends --assume-yes build-essential libsqlite3-0
```

Comment:
Why are you mixing `apt` and `apt-get`? :thinking: And how is sqlite related to `-race`?

Reply:
I can change it, it is the same just without dash `-`. I'll remove libsqlite3-0, i tihnk it is not needed.

Reply:
91d1e317838159a215d3022a178d036a56cb02f2